docs/docsite/rst/community/index.rst

```diff
@@ -8,8 +8,72 @@ Welcome to the Ansible Community Guide!
 
 The purpose of this guide is to teach you everything you need to know about being a contributing member of the Ansible community.
 
+All types of contributions are welcome, and necessary to Ansible's continued success.
+
 To get started, please read and understand the :ref:`code_of_conduct`, and then select one of the following topics.
 
+
+Getting Started
+===============
+
+* Read the :ref:`official documentation <ansible_documentation>`
+* I want to talk to other Ansible users, how do I find a :ref:`Ansible Meetup near me <ansible_community_meetup>`.
+* I have a question, what :ref:`Ansible email lists, IRC channels <communication>` are there?
+* How can I learn more - `books <https://www.ansible.com/resources/ebooks>`_, training courses, `videos <https://www.ansible.com/resources/videos>`_ - Not sure about training courses
+* How do I get notified when new Ansible versions are released? - Subscribe to `Ansible Announce list <https://groups.google.com/forum/#!forum/ansible-announce>`_.
+
+Understanding Ansible's development process
+===========================================
+
+* I found a bug - :ref:`how to report bugs <reporting_bugs>`
+* I need functionality Ansible doesn't offer - :ref:`requesting functionality <request_features>`.
+* If I'm not a programmer, can I contribute?
+
+   * Help us :ref:`improve Documentation <community_documentation_contributions>`.
+   * Join a :ref:`Working Groups, and how can I get involved? <community_working_groups>`.
+* How can I review Issues and Pull Requests?
+* :ref:`Ansible Development Cycle <community_development_process>`.
+* Understanding Ansible's release cycle:
+
+  * What releases are current - :ref:`Ansible Release status <release_schedule>`.
+  * `How do I get notified of releases <https://groups.google.com/forum/#!forum/ansible-announce>`.
+  * How do I request a bug fix gets backported - :ref:`create a backport PR <backport_process>`.
+  * What's planned for future Ansible Releases - see the :ref:`Ansible's Roadmaps <roadmaps>`.
+* Understanding Ansible's :ref:`Testing (CI) process<developing_testing>`.
+
```

Comment:
+ Other ways to get involved https://docs.ansible.com/ansible/devel/community/how_can_I_help.html